UM-USElab/gradecraft-development
# Upload inline images in Froala

### Status
**IN DEVELOPMENT**

### Description
* It should be possible for instructors and students to upload an inline image in the Froala editor. Currently uploading images does not work and is disabled. Now, uploading images is allowed and is uploaded to a new directory in the uploads folder (files/uploads/froala_uploads) and can be inserted as inline images in the editor.

### Related PRs
List related PRs against other branches:

branch | PR
------ | ------
image-button | #4235 

### Migrations
NO

### Steps to Test or Reproduce
1. As an instructor or student access a page that has the Froala editor and allows images to be uploaded. Click the image button in the editor and upload an image. 
2. The image that is uploaded should be inserted as an inline image in the editor and it should be possible to view and edit the uploaded image even after navigating away and then back to the page where the image was uploaded in.

### Impacted Areas in Application
* Uploading inline images in the Froala editor

======================
Closes #4210
